docs/docs/adrs/adr-019-permissionless-ics.md

```diff
@@ -171,7 +171,7 @@ message MsgCreateConsumer {
 }
 ```
 
-Note that `metadata` is a required field, while the `initialization_parameterrs` and `power_shaping_parameters` are optional and can later be set using `MsgUpdateConsumer`.
+Note that `metadata` is a required field, while the `initialization_parameters` and `power_shaping_parameters` are optional and can later be set using `MsgUpdateConsumer`.
```

Comment:
Took into account [older comments](https://github.com/cosmos/interchain-security/pull/2261#discussion_r1757125037) that were added after the PR got merged.